Exoft.Gamification.Api.Data.Core/Entities/Event.cs

```diff
@@ -8,16 +9,8 @@ public class Event : Entity
 
         public DateTime Time { get; set; }
```

Comment:
Time is not the best name. I'd use CreationTime, TimeCreated etc.
Also i'd try to incapsulate logic of setting time.... try to do private setter and set utc now in constructor... check if that works